user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_parameters/I-DOCSIS_parameters_node_segment.md

```diff
@@ -70,10 +70,30 @@ This page contains an overview of the Node Segment parameters available in the I
 
 - **Percentage CM Reflection OOS**: Calculated. The percentage of cable modems associated with the given service group that are affected by reflection.
 
-- **Number CM Group Delay or Reflection OOS**: Calculated. The number of cable modems associated with the given service group that are affected by NMTER being out of spec (OOS).
+- **Number CM Group Delay or Reflection OOS**: Calculated. The number of cable modems associated with the given service group that are affected by Pre-MTTER being out of spec (OOS) or Post-MTTER being out of spec (OOS).
+
+- **Percentage CM Group Delay or Reflection OOS**: Calculated. The percentage of cable modems associated with the given service group that are affected by Pre-MTTER being out of spec (OOS) or Post-MTTER being out of spec (OOS).
+
+> [!NOTE]
+> Note that **Percentage CM Group Delay OOS** and **Percentage CM Reflection OOS** parameters were afected by NMTTER being out of spec, this was changed since **Percentage CM Group Delay or Reflection OOS** did not have relation between the two previous ones.
```

Comment:
Is this related to RN 36344? Because if yes, I'd add this as follows:
```suggestion
> Note that prior to EPM I-DOCSIS 6.1.8<!-- RN 36344 -->, the **Percentage CM Group Delay OOS** and **Percentage CM Reflection OOS** parameters were affected by NMTTER being out of spec instead. This has been changed as **Percentage CM Group Delay or Reflection OOS** did not have relation between the two previous ones.
```

Reply:
It would be helpful, however I have done changes in the RN that belongs to 6.1.8. can you take a look, please?

Reply:
I have now added this release note to the PR based on the latest version on the intranet. Could you check if everything is OK?